packages/components/ovh-at-internet/src/ovh-at-internet.ts

```diff
@@ -308,10 +308,14 @@ export default class OvhAtInternet extends OvhAtInternetConfig {
   }
 
   trackPage(data: LegacyTrackingData): void {
+    const trackingData = data;
+    if (trackingData.page && !trackingData.page.name) {
+      trackingData.page = { name: trackingData.page.name };
```

Comment:
Also, it does not make sense to check `!trackingData.page.name` and then resetting the value to undefined/null/false as the code block is only executed when name does not exist on page.